ca/config.go

```diff
@@ -96,6 +97,22 @@ func (s *SecurityConfig) RootCA() *RootCA {
 	return s.rootCA
 }
 
+// KeyWriter returns the object that can write keys to disk
+func (s *SecurityConfig) KeyWriter() KeyWriter {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	return s.keyReadWriter
```

Comment:
Is `s.keyReadWriter` ever expected to change after creating the security config? If not, the locking seems unnecessary.